### DISCUSSION TITLE
Self Hosted Studio shows endless loading screen

### DISCUSSION THREAD

**Discussion by ravenbroetzmann** [supabase/supabase#7973] [Questions] [answered]
I did a `docker-compose up` in the docker dir with default `.env` from the `.env.example` configuration.

the kong api seems to work. But the supabase studio is stuck with a loading screen.

if I open the dev Tools 
I have one 404 error on `/undefined/auth/tokens`

Where is this `undefined` Coming from ?

thanks for your help

**Comment by silentworks** [comment, marked as answer]
You have to setup your JWT token stuff else it won't work as expected. Please follow the guide especially this step https://supabase.com/docs/guides/hosting/overview#api-keys

**Reply by ravenbroetzmann** [reply]
that's pretty obvious, thanks for you help
